specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -0,0 +1,692 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "./models.tsp";
+import "@azure-tools/typespec-autorest";
+
+using Azure.Core;
+using Azure.Core.Traits;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Autorest;
+
+namespace AzureLoadTesting;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@tag("Test")
+interface LoadTestAdministration {
+  @summary("Create a new test or update an existing test.")
+  @doc("Create a new test or update an existing test.")
+  @route("/tests/{testId}")
+  @patch
+  @example("./examples/CreateOrUpdateTest.json", "CreateOrUpdateTest")
+  createOrUpdateTest is Azure.Core.Foundations.Operation<
+    {
+      @header 
+      @doc("Content type.")
+      contentType: "application/merge-patch+json",
+      
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+
+      @doc("Load test model")
+      @body
+      body: Test;
+    },
+    ResourceCreatedOrOkResponse<Test>, {}, ErrorResponseBody
+  >;
+
+  @summary("Delete a test by its name.")
+  @doc("Delete a test by its name.")
+  @route("/tests/{testId}")
+  @delete
+  @example("./examples/DeleteTest.json", "DeleteTest")
+  deleteTest is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+    },
+    void, {}, ErrorResponseBody
+  >;
+
+  @summary("Get load test details by test name")
+  @doc("Get load test details by test name")
+  @route("/tests/{testId}")
+  @get
+  @example("./examples/GetTest.json", "GetTest")
+  getTest is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+    },
+    Test, {}, ErrorResponseBody
+  >;
+
+  @summary("""
+Get all load tests by the fully qualified resource Id e.g
+subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}.
+""")
+  @doc("""
+Get all load tests by the fully qualified resource Id e.g
+subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}.
+""")
+  @route("/tests")
+  @get
+  @example("./examples/ListTests.json", "ListTests")
+  listTests is Azure.Core.Foundations.Operation<
```

Comment:
This operation didn't fit into a ResourceList operation?

Reply:
Updated as suggested